.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -135,20 +118,6 @@ jobs:
       - run: node -v
       - run: yarn -v
 
-      - name: Get yarn cache dir
-        id: setup-yarn
-        shell: bash
-        run: |
-          echo "yarn-cache-dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-
-      - name: Yarn Cache
-        uses: actions/cache@v4
```

Comment:
why removing yarn cache? do they actually work? i remember trying to set it up long time ago but didn't work for some reason

Reply:
In the unit tests they have no performance diff still -- the compilation is implicit to yarn test

Reply:
there are two caches, npm packages and transpiled/optimized javascript. does cache for npm packages work in this workflow?

Reply:
You would expect so, but it doesn't look like it. Bootstrap also doesn't have a noticeable speedup by caching packages.

Reply:
yeah i think i saw similar results when i tried it, not sure why